fieldmanager.php

```diff
@@ -499,6 +499,23 @@ function fm_sanitize_textarea( $value ) {
 }
 
 /**
+ * Call wp_die() with an error message for Fieldmanager_Field::_failed_validation().
+ *
+ * @since 1.1.0
+ *
+ * @param string $error_message Optional. Error message in addition to the
+ *                              default message for validation errors.
+ */
+function fm_failed_validation_to_wp_die( $error_message = '' ) {
+	wp_die( esc_html(
+		$error_message
+		. "\n\n"
+		. __( "You may be able to use your browser's back button to resolve this error.", 'fieldmanager' )
+	) );
+}
+add_action( 'fm_failed_validation', 'fm_failed_validation_to_wp_die' );
```

Comment:
I built a pretty cool admin notice framework for Apple News that we could look at using. It uses user meta instead of transients, but kind of amounts to the same. It doesn't require a query arg to trigger.

I like #2. I think it would be cool if it didn't save the data but displayed the invalid values in the admin notice. I'd also consider CSS to highlight the field(s) that had invalid values in red. They should display their values prior to the last save attempt though and not the invalid values.